core/models.py

```diff
@@ -143,6 +143,9 @@ def count_descendants(post: "Post") -> int:
 
         return count_descendants(self)
 
+    def is_saved_by(self: "Post", user: User) -> bool:
+        return SavedPost.objects.filter(user=user, post=self).exists()

```

Comment:
What does this do? It is a misleading name, I would expect it to return the name who saved the post, not boolean. 
Please change naming.